apps/hash-api/src/graphql/resolvers/knowledge/entity/entity.ts

```diff
@@ -489,3 +492,34 @@ export const isEntityPublicResolver: ResolverFn<
     entityId,
   });
 };
+
+export const getEntityAuthorizationRelationshipsResolver: ResolverFn<
+  Promise<EntityAuthorizationRelationship[]>,
```

Comment:
`ResolverFn` already wraps the result in a `Promise` (noticed since I started typing these)
```suggestion
  EntityAuthorizationRelationship[],
```